tests/week-2/c-comparison-operators.test.js

```diff
@@ -0,0 +1,34 @@
+const { importSourceAndRead } = require("../helpers");
+require("../../week-2/C-comparison-operators/exercise.js");
+
+let contents = "";
+
+importSourceAndRead(
+  require,
+  "../../week-2/C-comparison-operators/exercise.js"
+).then(res => (contents = res));
+
+test("Comparison statement is written correctly for 'moreStudentsThanMentors'", () => {
+  const hasCorrectKeywords = !!contents.match(
+    /moreStudentsThanMentors\s*=\s*studentCount\s*>\s*mentorCount/
+  );
+  expect(hasCorrectKeywords).toBe(true);
+});
+
+test("Comparison statement is written correctly for 'enoughSpaceInRoom'", () => {
+  const hasCorrectKeywords =
+    !!contents.match(
+      /enoughSpaceInRoom\s*=\s*roomMaxCapacity\s*>=?\s*studentCount\s*\+\s*mentorCount/
```

Comment:
super good test! one of the issues we had last year was that students were looking at the values of the mentorCount and studentCount variables and then manually writing:
`var enoughSpaceInRoom = true` instead of the correct `var enoughSpaceInRoom = roomMaxCapacity >= studentCount + mentorCount`

...which was console.logging the correct value :))

This test will help us spot that though!